README.md

```diff
@@ -1 +1,46 @@
-"# math-magicians" 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+> "Math magicians" is a website for all fans of mathematics. It allows users to:
+>
+> - Make simple calculations.
+> - Read a random math-related quote.
+
+## Built With
+
+- React
+
+## Live Demo
+
+Not yet
+
+## Getting Started
+
+- Clone this repo with:
+`git clone https://github.com/fasaldana/math-magicians.git`
+
+- Run application using:
+`npm start`
```

Comment:
- [ ] The proper way to put the instruction to clone and run the project locally is to mention all the steps so that new users can easily run your project who don't have this background.

```
$ git clone https://github.com/fasaldana/math-magicians.git
$ cd math-magicians
$ npm install
$ npm start
```

If we miss any step here, it will produce error so it's better to put all in one place.